pkg/lclient/fakeclient.go

```diff
@@ -39,6 +39,55 @@ var mockImageSummary = []types.ImageSummary{
 	},
 }
 
+var mockContainerJSONList = []types.ContainerJSON{
+	types.ContainerJSON{
+		&types.ContainerJSONBase{
+			Name:  "/node",
+			Image: "node",
+			ID:    "1",
+		},
+		[]types.MountPoint{
+			{
+				Destination: OdoSourceVolumeMount,
+			},
+		},
+	},
```

Comment:
You need to mention the type
```suggestion
	{
		ContainerJSONBase: &types.ContainerJSONBase{
			Name:  "/node",
			Image: "node",
			ID:    "1",
		},
		Mounts: []types.MountPoint{
			{
				Destination: OdoSourceVolumeMount,
			},
		},
	},
```

Reply:
It helps get past `too few values in types.ContainerJSON literal` error. I had removed the type info because I got `mixture of field:value and value initializers` error when I had mentioned type for one field but not for another.